### PR TITLE
Add instructions for rebasing a fork

### PR DESCRIPTION
**Pull request by izgeri**
Add instructions for rebasing a fork against the upstream master branch

### CODE REVIEW

File: Conjur/conventions/git-tips-and-tricks.md [hunk 1]
```diff
@@ -165,6 +167,29 @@ git push --force
...
+
+Fetch all the branches of the remote upstream project:
+```
+git fetch upstream
+```
+
+Rewrite your local branch to replay any commits from your local branch on top of the `upstream/master`:
+```
+git rebase upstream/master
+```
```
Comment (sgnn7) [inline Conjur/conventions/git-tips-and-tricks.md:186]: I'd add here to do a `git log` just to make sure the branch looks as-expected before overwriting the master in our repo

**Review by sgnn7 — CHANGES_REQUESTED** [review]
@izgeri Just one tiny comment

File: Conjur/conventions/git-tips-and-tricks.md (hunk 1 above)
Comment (izgeri) [reply Conjur/conventions/git-tips-and-tricks.md:186]: do you mean "before overwriting the master in your repo"?

Comment (sgnn7) [reply Conjur/conventions/git-tips-and-tricks.md:186]: @izgeri correct. after the rebase but before the push.

File: Conjur/conventions/git-tips-and-tricks.md [hunk 2]
```diff
@@ -165,6 +167,29 @@ git push --force
...
+
```
Comment (izgeri) [inline Conjur/conventions/git-tips-and-tricks.md:187]:
```suggestion

Review the git logs before publishing these changes to ensure the git history looks as expected:
```
git logs
```

```

File: Conjur/conventions/git-tips-and-tricks.md [hunk 3]
```diff
@@ -165,6 +167,34 @@ git push --force
...
+Review the git logs before publishing these changes to ensure the git history looks as expected:
+```
+git logs
```
Comment (izgeri) [inline Conjur/conventions/git-tips-and-tricks.md:190]:
```suggestion
git log
```

File: Conjur/conventions/git-tips-and-tricks.md (hunk 1 above)
Comment (izgeri) [reply Conjur/conventions/git-tips-and-tricks.md:186]: fixed - will need a squash merge

**Review by sgnn7 — APPROVED** [review]

**Pull request merged**